### PR TITLE
Remove ActiveAdmin

### PR DESCRIPTION
**Pull request by atzorvas**
Implements #132

### CODE REVIEW

**Comment by AEgan** [comment]
👏  👏  Love the Red PRs

We can also remove `activeadmin` and `active_skin` from the [Gemfile](https://github.com/rubyforgood/share_christmas/blob/2276f14eca543adbba0761b1db59fc1768366361/Gemfile#L6-L7) 

It's probably better to create a migration to drop the activeadmin tables instead of deleting the migrations that create them, but at this stage I'm not sure we need to be super careful with database changes

**Comment by atzorvas** [comment]
@AEgan I could go with restoring the migrations and adding a removing one.

Though if we don't have this deployed somewhere already, maybe I would keep it as it is.
I'm ok with both

**Comment by AEgan** [comment]
I deployed it to Heroku during ruby for good to make sure we had everything set up to be able to deploy. I/we haven't been actively deploying with every merge and keeping a production environment up to date

**Comment by atzorvas** [comment]
@AEgan reverted the migrations.

We had one to create an active_admin_comments table, and one to remove it so it seems that we don't have to introduce another one.

File: .env [hunk 1]
```diff
@@ -1,5 +1,7 @@
-export ADMIN_EMAIL=admin@example.com
-export ADMIN_PASSWORD=password
+export VC_ADMIN_EMAIL=admin@vc.com
+export VC_ADMIN_PASSWORD=password
+export ORG_ADMIN_EMAIL=admin@org.com
+export ORG_ADMIN_PASSWORD=password
```
Comment (alexcameron89) [inline .env]: The [README](https://github.com/rubyforgood/share_christmas/blob/master/README.md#quick-start) will need updated usernames for this. I love this commit, the Activeadmin stuff was confusing me a little.

**Comment by atzorvas** [comment]
@AEgan I think this should be good to go now /cc @craig-riecke

**Comment by craig-riecke** [comment]
👍 I agree - red PR's are good!  Thanks @atzorvas.  Merging...

**Pull request merged**